src/Components/ServiceScene/Breakdowns/FieldValuesBreakdownScene.tsx

```diff
@@ -38,12 +38,34 @@ export interface FieldValuesBreakdownSceneState extends SceneObjectState {
   $data?: SceneDataProvider;
   lastFilterEvent?: AddFilterEvent;
 }
+
 export class FieldValuesBreakdownScene extends SceneObjectBase<FieldValuesBreakdownSceneState> {
   constructor(state: Partial<FieldValuesBreakdownSceneState>) {
     super(state);
     this.addActivationHandler(this.onActivate.bind(this));
   }
 
+  public static Selector({ model }: SceneComponentProps<FieldValuesBreakdownScene>) {
```

Comment:
auto-format moved the static methods up top, I can revert this if we want to keep lines changed down